moment.parseFormat.js

```diff
@@ -4,213 +4,212 @@
   /* global define */
```

Comment:
Or maybe better even do it like the PouchDB folks do it in their plugins:
https://github.com/nolanlawson/pouchdb-all-dbs/blob/master/index.js#L148-L151